x-pack/plugins/uptime/public/pages/overview.tsx

```diff
@@ -12,14 +12,15 @@ import styled from 'styled-components';
 import { useBreadcrumbs } from '../hooks/use_breadcrumbs';
 import { useTrackPageview } from '../../../observability/public';
 import { MonitorList } from '../components/overview/monitor_list/monitor_list_container';
-import { EmptyState, FilterGroup } from '../components/overview';
+import { EmptyState } from '../components/overview';
 import { StatusPanel } from '../components/overview/status_panel';
 import { QueryBar } from '../components/overview/query_bar/query_bar';
 import { MONITORING_OVERVIEW_LABEL } from '../routes';
+import { FilterGroup } from '../components/overview/filter_group/filter_group';
 
 const EuiFlexItemStyled = styled(EuiFlexItem)`
   && {
-    min-width: 598px;
+    min-width: 800px;
```

Comment:
Not sure why this change? Perhaps to help account for added size of the new filter component?